docs/content/preview/yugabyte-voyager/migrate/live-fall-forward.md

```diff
@@ -0,0 +1,573 @@
+
+After making the manual changes, re-run the `yb-voyager analyze-schema` command. This generates a fresh report using your changes. Repeat these steps until the generated report contains no issues.
+
+To learn more about modelling strategies using YugabyteDB, refer to [Data modeling](../../reference/data-modeling/).
+
+{{< note title="Manual schema changes" >}}
+
+- `CREATE INDEX CONCURRENTLY` is not currently supported in YugabyteDB. You should remove the `CONCURRENTLY` clause before trying to import the schema.
+
+- Include the primary key definition in the `CREATE TABLE` statement. Primary Key cannot be added to a partitioned table using the `ALTER TABLE` statement.
+
+{{< /note >}}
+
+### Import schema
+
+Import the schema using the `yb-voyager import schema` command.
+
+{{< note title="Usage for target_db_schema" >}}
+
+`yb-voyager` imports the source database into the `public` schema of the target database. By specifying `--target-db-schema` argument during import, you can instruct `yb-voyager` to create a non-public schema and use it for the schema/data import.
+
+{{< /note >}}
+
+An example invocation of the command is as follows:
+
+```sh
+# Replace the argument values with those applicable for your migration.
+yb-voyager import schema --export-dir <EXPORT_DIR> \
+        --target-db-host <TARGET_DB_HOST> \
+        --target-db-user <TARGET_DB_USER> \
+        --target-db-password <TARGET_DB_PASSWORD> \ # Enclose the password in single quotes if it contains special characters..
+        --target-db-name <TARGET_DB_NAME> \
+        --target-db-schema <TARGET_DB_SCHEMA>
+```
+
+Refer to [import schema](../../reference/yb-voyager-cli/#import-schema) for details about the arguments.
+
+yb-voyager applies the DDL SQL files located in the `$EXPORT_DIR/schema` directory to the target database. If yb-voyager terminates before it imports the entire schema, you can rerun it by adding the `--ignore-exist` option.
+
+{{< note title="Importing indexes and triggers" >}}
+
+Because the presence of indexes and triggers can slow down the rate at which data is imported, by default `import schema` does not import indexes and triggers. You should complete the data import without creating indexes and triggers. Only after data import is complete, create indexes and triggers using the `import schema` command with an additional `--post-import-data` flag.
+
+{{< /note >}}
+
+### Export and import schema to fall-forward database
+
+Manually, setup the fall-forward database with the same schema as that of the source database with the following considerations:
+
+- The table names on the fall-forward database need to be case insensitive ( as YB Voyager currently does not support case-sensitivity.)
+- Do not create indexes and triggers at the schema setup stage, as it will degrade performance of importing data into the fall-forward database. Create them later as mentioned in Step 4 of [Fall-forward switchover](#fall-forward-switchover-optional).
```

Comment:
```suggestion
- Do not create indexes and triggers at the schema setup stage, as it will degrade performance of importing data into the fall-forward database. Create them later as described in [Fall-forward switchover](#fall-forward-switchover-optional).
```